### PR TITLE
fix: broken link maintenance

### PR DESCRIPTION
**Pull request by mcdurdin**
Relates to #233.

Fixes most remaining broken links in help.keyman.com except for older keyboard documentation versions; this coming next.

### CODE REVIEW

File: developer/8.0/docs/reference_kmw20_ui_button.php [hunk 1]
```diff
@@ -6,10 +6,11 @@
...
-    The simplest user interface is the "button" UI, which appears at a fixed position on the web page, 
+    The simplest user interface is the "button" UI, which appears at a fixed position on the web page,
     and is attached to an empty 'Div' element with <span class="emphasis"><em>id='KeymanWebControl'</em></span> added to the page where appropriate.
-  </p><p> 
-    When the mouse is hovered over the 'button' UI, a menu of supported input languages and keyboards is shown: 
-  </p><span class="inlinemediaobject"><img src="developer_images/kmw20_ui_button.png"></span><p>
+  </p><p>
+    When the mouse is hovered over the 'button' UI, a menu of supported input languages and keyboards is shown:
+  </p><span class="inlinemediaobject"><img src="/developer/engine/web/2.0/reference/ui/images/ui_button.png"></span><p>
```
Comment (darcywong00) [inline developer/8.0/docs/reference_kmw20_ui_button.php:13]: I noticed this PR has similarities to #229 that I did for `staging`. There, I used `../../engine/web/images/ui-button.gif`

Comment (mcdurdin) [reply developer/8.0/docs/reference_kmw20_ui_button.php:13]: I recalled that but couldn't remember where you'd done it; I wasn't too worried about the actual file, so long as it was correct :)

File: developer/core/desktop/web.config [hunk 2]
```diff
@@ -6,7 +6,7 @@
                 <!-- Desktop engine content hasn't changed between 11.0 and 13.0 -> 11.0 -->
                 <rule name="Redirect 13.0, 12.0/ to 11.0/">
```
Comment (darcywong00) [inline developer/core/desktop/web.config:7]:
```suggestion
                <rule name="Redirect 13.0, 12.0/ to current-version/">
```

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**